Original patch from gentoo: ../../dev/crosstool-NG/ct-ng.trunk/patches/strace/4.5.17/110-dont-use-REG_SYSCALL-for-sh.patch
-= BEGIN original header =-
diff -dur strace-4.5.16.orig/process.c strace-4.5.16/process.c
-= END original header =-
diff -durN strace-4.5.17.orig/process.c strace-4.5.17/process.c
--- strace-4.5.17.orig/process.c	2008-07-18 04:16:47.000000000 +0200
+++ strace-4.5.17/process.c	2008-10-24 21:30:13.000000000 +0200
@@ -2726,7 +2726,6 @@
        { 4*REG_GBR,            "4*REG_GBR"                             },
        { 4*REG_MACH,           "4*REG_MACH"                            },
        { 4*REG_MACL,           "4*REG_MACL"                            },
-       { 4*REG_SYSCALL,        "4*REG_SYSCALL"                         },
        { 4*REG_FPUL,           "4*REG_FPUL"                            },
        { 4*REG_FPREG0,         "4*REG_FPREG0"                          },
        { 4*(REG_FPREG0+1),     "4*REG_FPREG1"                          },
